solutions/cpp/01-oo8/code/src/main.cpp

```diff
@@ -6,7 +6,7 @@ int main() {
   std::cerr << std::unitbuf;
 
   std::cout << "$ ";
-
+  
```

Comment:
@andy1li let's remove trailing spaces

Reply:
Look like it's actually for indentation. Without it the diff will behave weirdly.

<img width="833" alt="image" src="https://github.com/user-attachments/assets/8349efd7-c8f5-478c-9109-8e881db690ff" />

Reply:
@andy1li yep, but for empty lines we don't need indentation (I don't think we have it for the other lines)

Reply:
Ah, got it! (The weird diff came from the stage 2 solution. Changing that as well.)